src/main/kotlin/lotto/model/Lottos.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.model
+
+/**
+ * 로또 당첨번호 및 통계 결과 클래스
+ * */
+class Lottos private constructor(private val price: Price, private val purchasedLotto: List<Lotto>, private val lastWinNumber: Lotto) {
+    fun compareLottoList(): LottoStatisticFormat {
+        val hashMap = hashMapOf<LottoRank, Int>()
+        purchasedLotto.forEach { lotto ->
+            val rank = LottoRank.compareMatchRank(compareNumber(lotto))
+            hashMap[rank] = hashMap.getOrDefault(rank, 0) + 1
+        }
+        return LottoStatisticFormat(price, hashMap)
+    }
+
+    private fun compareNumber(item: Lotto): Int {
+        var number = 0
+        item.numbers
+            .forEach { lottoNumber ->
+                if (lastWinNumber.numbers.contains(lottoNumber)) ++ number
```

Comment:
```suggestion
                if (lastWinNumber.numbers.contains(lottoNumber)) ++number
```
++연산자는 띄어쓰기 없이 붙여주면 좋을 것 같아요